bliss/encoder.py

```diff
@@ -114,15 +115,14 @@ def dist_param_groups(self):
             d[gal_flux] = UnconstrainedLogNormal()
         return d
 
-    def _get_num_input_channels(self):
+    def _get_num_images(self):
         """Determine number of input channels for model based on desired input transforms."""
-        num_channels = 2  # image + background
+        num_imgs = 2  # img + bg
```

Comment:
`num_channels` makes more sense to me; is there something wrong with it?

Reply:
(I wouldn't necessarily thing of the background as a distinct image.)

Reply:
Oh I see, this is the length of the third axis below, whereas channels is the second axis. Is there another name for it we can use? (`num_imgs` seems like it could refer to any axis.)

Reply:
I think in the typical sense, we have `num_channels` set to the number of bands (i.e. in line 90). `num_imgs` definitely isn't the correct term but it is whatever we want to call the third dimension in the 5-D input containing all band-specific things (e.g. image, background, deconvolved image, PSF parameters, etc.)

Definitely open to a better name - just think that `num_channels` should be reserved for the number of bands since the actual number of 'input channels' we pass into the encoder corresponds with the number of bands.

Reply:
I'm struggling to think of an ideal name - would `num_features` be more appropriate? Still don't think it's great..

Reply:
`num_features_per_band` seems OK ... is that too long?

Reply:
It's better than `num_imgs` for sure - think that definitely is descriptive enough. I like it